root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -1713,6 +1713,48 @@ test('URL cleanup component: auto-select, clean-up, and validation', {}, functio
                              input_url: 'https://commons.m.wikimedia.org/wiki/File:Karel-R%C5%AF%C5%BEi%C4%8Dka-star%C5%A1%C3%AD.jpg#mw-jump-to-license',
                     expected_clean_url: 'https://commons.wikimedia.org/wiki/File:Karel-R%C5%AF%C5%BEi%C4%8Dka-star%C5%A1%C3%AD.jpg',
         },
+        {
+                             input_url: 'https://commons.wikimedia.org/wiki/Category:Opeth#/media/File:Opeth_-_Kavarna_Rock_Fest_2011.jpg',
+                     input_entity_type: 'artist',
+            expected_relationship_type: 'image',
+                    expected_clean_url: 'https://commons.wikimedia.org/wiki/File:Opeth_-_Kavarna_Rock_Fest_2011.jpg',
+        },
+        {
+                             input_url: 'http://commons.wikimedia.org/wiki/Category:Michl_M%C3%BCller?uselang=de#/media/File:Michl_M%C3%BCller_Garitz_2013.JPG',
+                     input_entity_type: 'artist',
+            expected_relationship_type: 'image',
+                    expected_clean_url: 'https://commons.wikimedia.org/wiki/File:Michl_M%C3%BCller_Garitz_2013.JPG',
+        },
+        {
+                             input_url: 'https://commons.wikimedia.org/wiki/File%3APolygram_Gold_Disc_Award_Wet_Wet_Wet_Ralph_Ruppert_1987.jpg',
+                    expected_clean_url: 'https://commons.wikimedia.org/wiki/File:Polygram_Gold_Disc_Award_Wet_Wet_Wet_Ralph_Ruppert_1987.jpg',
+        },
+        {
+                             input_url: 'https://commons.wikimedia.org/wiki/Within_Temptation',
+                                        // gallery page
+                     input_entity_type: 'artist'.
```

Comment:
There it is, a period at the end instead of a comma. :) But I ran the tests again after fixing that locally and there are some more issues:

```
# input URL [291] = https://commons.wikimedia.org/wiki/Within_Temptation
not ok 650 Test is invalid: "input_entity_type" is specified without "expected_relationship_type".
  ---
    operator: fail
  ...
# input URL [292] = https://commons.wikimedia.org/wiki/Category:Ewa_Demarczyk
not ok 651 Test is invalid: "input_entity_type" is specified without "expected_relationship_type".
  ---
    operator: fail
  ...
# input URL [293] = https://commons.wikimedia.org/w/index.php?search=umberto+alongi&title=Special%3ASearch&go=Go
not ok 652 Test is invalid: "input_entity_type" is specified without "expected_relationship_type".
  ---
    operator: fail
  ...
```